listenbrainz/webserver/templates/art/svg-templates/yim-2022-albums.svg

```diff
@@ -50,15 +49,15 @@
             </text>
         </g>
         <g>
-            <image class="album-art" x="265" y="250" xlink:href="{{ image_urls[0] }}" />
-            <image class="album-art" x="265" y="385" xlink:href="{{ image_urls[1] }}" />
-            <image class="album-art" x="265" y="520" xlink:href="{{ image_urls[2] }}" />
-            <image class="album-art" x="400" y="250" xlink:href="{{ image_urls[3] }}" />
-            <image class="album-art" x="400" y="385" xlink:href="{{ image_urls[4] }}" />
-            <image class="album-art" x="400" y="520" xlink:href="{{ image_urls[5] }}" />
-            <image class="album-art" x="535" y="250" xlink:href="{{ image_urls[6] }}" />
-            <image class="album-art" x="535" y="385" xlink:href="{{ image_urls[7] }}" />
-            <image class="album-art" x="535" y="520" xlink:href="{{ image_urls[8] }}" />
+            <image class="album-art" preserveAspectRatio="xMidYMid slice" x="265" y="250" xlink:href="{{ image_urls[0] }}" />
+            <image class="album-art" preserveAspectRatio="xMidYMid slice" x="265" y="385" xlink:href="{{ image_urls[1] }}" />
+            <image class="album-art" preserveAspectRatio="xMidYMid slice" x="265" y="520" xlink:href="{{ image_urls[2] }}" />
+            <image class="album-art" preserveAspectRatio="xMidYMid slice" x="400" y="250" xlink:href="{{ image_urls[3] }}" />
+            <image class="album-art" preserveAspectRatio="xMidYMid slice" x="400" y="385" xlink:href="{{ image_urls[4] }}" />
+            <image class="album-art" preserveAspectRatio="xMidYMid slice" x="400" y="520" xlink:href="{{ image_urls[5] }}" />
+            <image class="album-art" preserveAspectRatio="xMidYMid slice" x="535" y="250" xlink:href="{{ image_urls[6] }}" />
+            <image class="album-art" preserveAspectRatio="xMidYMid slice" x="535" y="385" xlink:href="{{ image_urls[7] }}" />
+            <image class="album-art" preserveAspectRatio="xMidYMid slice" x="535" y="520" xlink:href="{{ image_urls[8] }}" />
             <rect x="235" y="170" width="452" height="521" fill="url(#pattern2)"/>
```

Comment:
I made a couple of changes in this SVG file in commit 7a527c0ce31ce6c25faa14d2a7a61e14757c03c0 that will need to be ported over/merged. Automatic merge might work, but I thought I'd mention it.

Reply:
Not sure if these changes are still applicable? Doesn't look like there are any changes to this file anymore.

Reply:
Yup, all resolved.